libs/java/service/src/main/java/ca/bc/gov/tno/services/models/AuditColumns.java

```diff
@@ -0,0 +1,161 @@
+package ca.bc.gov.tno.services.models;
+
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import ca.bc.gov.tno.services.converters.Settings;
+import ca.bc.gov.tno.services.converters.ZonedDateTimeDeserializer;
+
+/**
+ * AuditColumns abstract class, provides standardized audit columns for entities
+ * in the DB.
+ */
+public abstract class AuditColumns implements Serializable {
+  /**
+   * The uid that identifies the user who created the record.
+   */
+  private UUID createdById = new UUID(0L, 0L);
+
+  /**
+   * A name that identifies the user who created the record.
+   */
+  private String createdBy = "";
+
+  /**
+   * When the record was created.
+   * Automatically set by the DB.
+   */
+  // @JsonFormat(shape = JsonFormat.Shape.STRING, pattern =
+  // Settings.dateTimeFormat)
+  // @JsonDeserialize(using = ZonedDateTimeDeserializer.class)
+  private String createdOn = ZonedDateTime.now().format(DateTimeFormatter.ofPattern(Settings.dateTimeFormat));
```

Comment:
Had to change all dates to strings because Java is horrible and can't handle standard date formats.